src/server/migrations/1_create_initial_tables.sql

```diff
@@ -14,5 +14,13 @@ CREATE TABLE edge (
   target_id INTEGER REFERENCES node(id)
 );
 
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  username TEXT NOT NULL,
+  password TEXT NOT NULL,
+  email TEXT NOT NULL,
+  UNIQUE (username, email)
```

Comment:
This makes the `username, email` pair unique, not either value, ie. you can register both `username=foo, email=foo@example.com()` and `username=foo, email=bar@example.com`.

Better written as `username TEXT NOT NULL UNIQUE` etc.